### PR TITLE
Flag Page

### PR DESCRIPTION
**Pull request by Rafalzie**
- Created microphone and Back Icon
-Flag Page

### CODE REVIEW

File: src/components/BackArrow.tsx
[the remark this answers is not in the document]
Comment (utlandingur) [reply src/components/BackArrow.tsx:1]: And remove this component entirely

File: src/components/BackIcon.tsx
Comment (utlandingur) [inline src/components/BackIcon.tsx:1]: Doesnt do anything. And as mentioned before, this isnt needed. Just use the icon in the child of a button

File: src/components/FlagCard.tsx
Comment (utlandingur) [inline src/components/FlagCard.tsx:1]: remove empty object

Comment (utlandingur) [reply src/components/FlagCard.tsx:1]: Don't need a margin bottom.
Handle the spacing in the parent component

File: src/components/microphone.tsx
Comment (utlandingur) [inline src/components/microphone.tsx:1]: Remove the bad import from microphone and just use the absolute path (src/...)

Comment (utlandingur) [reply src/components/microphone.tsx:1]: Don't remove the outline. Just maybe change color or something.

Comment (utlandingur) [reply src/components/microphone.tsx:1]: I would probably remove this as a component.

File: src/main.tsx
Comment (utlandingur) [inline src/main.tsx:1]: undo this change

**Review by utlandingur — DISMISSED** [review]
Talked through
Hope this helps

**Review by Evelina-Ok — APPROVED** [review]
